packages/language-server/src/rename/renameUtil.ts

```diff
@@ -16,17 +16,28 @@ function extractFirstWord(line: string): string {
   return line.replace(/ .*/, '')
 }
 
+function getType(currentLine: string): string {
+  const wordsInLine: string[] = currentLine.split(/\s+/)
+  return wordsInLine[1].replace('?', '').replace('[]', '')
```

Comment:
Does get `getAllRelationNames` function ensure that `getType` is only called for a relation field? 

`wordsInLine[1]` is unsafe and should be handled, in case this is accidently called on `@@id` line or some line that doesn't have 2 words. This would throw.

Reply:
good point, I added checks for all of those things 👍

Reply:
`getType` is called for all kinds of fields, not just for a relation field.